### PR TITLE
Changed the snatpolicy label key with UID

### PR DESCRIPTION
**Pull request by Vikash082**

### CODE REVIEW

**Review by anmol372 — CHANGES_REQUESTED** [review]
Hi Vikash,
I think the delete handler also needs the label key change:
[`func (agent *HostAgent) handleObjectDeleteForSnat(obj interface{})`](https://github.com/noironetworks/aci-containers/pull/993/files#diff-4e49a9da11f533bc9439f0b0a8da56512a9034875f67b8670cc1946497ecc949R1174-R1179)

File: pkg/hostagent/snats.go [hunk 1]
```diff
@@ -353,7 +353,8 @@ func (agent *HostAgent) handleSnatUpdate(policy *snatpolicy.SnatPolicy) {
 		for _, service := range services {
 			uids, _ := agent.getPodsMatchingObject(service, policy.ObjectMeta.Name)
 			poduids = append(poduids, uids...)
-			key, err := cache.MetaNamespaceKeyFunc(service)
+			// key, err := cache.MetaNamespaceKeyFunc(service)
```
Comment (snaiksat) [inline pkg/hostagent/snats.go:356]: pl remove this

**Review by snaiksat — COMMENTED** [review]
For consistency can we update L984 also?

Also, please squash the commits.

File: pkg/hostagent/snats.go (hunk 1 above)
Comment (Vikash082) [reply pkg/hostagent/snats.go:356]: Will address above comment & removed the commented code.

**Review by anmol372 — APPROVED** [review]

**Review by akhilamohanan — APPROVED** [review]

**Pull request merged**